spec/app.spec.js

```diff
@@ -0,0 +1,58 @@
+const assert = require('assert');
+const { cleanup, run } = require("./runner");
+const fs = require('fs');
+const os = require('os');
+
+describe('Test NeutralinoApp Class', function () {
+  before(() => {
+    run('neu create test-app');
+    process.chdir('test-app');
+  });
+
+  it('Test init() command', function () {
+    const demoJsContent = `
+    const NeutralinoApp = require("../../src/index.js");
+    
+    const app = new NeutralinoApp({
+      url: '/',
+      modes: {
+        window: {
+          width: 500,
+          height: 500
+        }
+      }
+    });
+    
+    app.init();
+    
+    setTimeout(() => {
+      process.exit(0)
+    }, 3000);
+    `;
+
+    fs.writeFileSync('demo.js', demoJsContent);
+
+    const command = `node demo.js`;
+    const result = run(command);
+
+    const binaryNames = {
+      linux: ['neutralino-linux_arm64', 'neutralino-linux_armhf', 'neutralino-linux_x64'],
+      darwin: ['neutralino-mac_arm64', 'neutralino-mac_universal', 'neutralino-mac_x64'],
+      win32: ['neutralino-win_x64.exe']
+    };
+
+    const expectedOutputPattern = "Starting process: {binaryName}  --load-dir-res --path=. --export-auth-info --neu-dev-extension  --url=/";
```

Comment:
Seems like you followed the framework spec implementation approach where we use Js code as a string. We used that approach since we deal with two Js environments there (webview and Node). But, in this scenario, we can directly import the `NodeNeutralino` class of this repo itself within test cases since we only work with the Node.js runtime. Please check the following pseudo-codes to learn more about this pattern:

```js
  const NeutralinoApp = require("../../src/index.js");
it('Test the init() method', function () {
    const app = new NeutralinoApp({
      url: '/',
      modes: {
        window: {
          width: 500,
          height: 500
        }
      }
    });
    
    app.init();
    app.exit();
```

Here is another pesudo-code:

```js
  const NeutralinoApp = require("../../src/index.js");
it('Test the app.getConfig() method', function () {
    const app = new NeutralinoApp({
      url: '/',
      modes: {
        window: {
          width: 500,
          height: 500
        }
      }
    });
    
    app.init();
    assert.ok(typeof app.getConfig() == 'object' );
    // More assertions go here as usual 
```

Please let me know if is idea is not clear -- we can discuss further about this on Discord :tada: